src/main/java/seedu/address/logic/Logic.java

```diff
@@ -30,11 +30,15 @@
      */
     ReadOnlyAddressBook getAddressBook();
 
-    /** Returns an unmodifiable view of the filtered list of persons */
-    ObservableList<Person> getFilteredPersonList();
+    /**
+     * Returns an unmodifiable view of the filtered list of passengers.
+     **/
+    ObservableList<Passenger> getFilteredPassengerList();
 
-    /** Returns an unmodifiable view of the filtered list of persons whether it is assigned to a driver already*/
-    ObservableList<Person> getFilteredPersonListByDriverStatus(Boolean isAssigned);
+    /**
+     * Returns an unmodifiable view of the filtered list of passengers whether it is assigned to a driver already.
```

Comment:
same goes for here as per above

Reply:
closing as upon discussion, we agreed that it's alright as checkstyle tests have passed it